opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -104,10 +104,21 @@ enum SeverityNumber {
   SEVERITY_NUMBER_FATAL4 = 24;
 }
 
-// Masks for LogRecord.flags field.
+// LogRecordFlags is defined as a protobuf 'uint32' type and is to be used as
+// bit-fields. Each non-zero value defined in this enum is a bit-mask.
+// To extract the bit-field, for example, use an expression like:
+//
+//   (logRecord.flags & LOG_RECORD_FLAG_TRACE_FLAGS_MASK)
+//
 enum LogRecordFlags {
```

Comment:
LogRecordFlags -> LogRecordFlag or the values get the "S"?

Reply:
We use plural for enum name (it contains many flags) and singular for each item in the enum. I think it is reasonable, so let's keep it that way and avoid more breaking changes than we need to.

Reply:
Then what about `LOG_RECORD_FLAG[S]_TRACE_FLAGS_MASK` to be consistent with other enums?

Reply:
Filed an issue to look at it: https://github.com/open-telemetry/opentelemetry-proto/issues/473